docs/platforms/dotnet/common/logs/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Set Up Logs
+sidebar_title: Logs
+description: "Structured logs allow you to send, view and query logs sent from your applications within Sentry."
+sidebar_order: 5600
```

Comment:
question: order

We're inconsistent on the order for SDKs that have both `Crons` and `Logs`
- `Logs` before `Crons`: Go, PHP
- `Crons` before `Logs`: Java, Python, Ruby

Where should we put it?
Does it matter?
@AbhiPrasad

Reply:
Let's not worry about this for now, I'll just merge what we have here.

I raised a thread in slack to chat more about this: https://sentry.slack.com/archives/C081M1KEQ0L/p1754917568834039. Once we make a decision there, I'll leave another comment here about what we are going to change. I'll own fixing the order here.

Reply:
will keep as is ... thanks!